spec/output/generator_spec_go/test.x/MyXDR_generated.go

```diff
@@ -411,7 +411,7 @@ var _ xdrType = (*Hash)(nil)
 //
 type Hashes1 [12]Hash
 // EncodeTo encodes this value using the Encoder.
-func (s *Hashes1) EncodeTo(e *xdr.Encoder) error {
+func (s Hashes1) EncodeTo(e *xdr.Encoder) error {
```

Comment:
This is wrong, we should be using a pointer here. But not when the synonym is a slice

Reply:
Will fix